CHANGELOG.md

```diff
@@ -1,7 +1,71 @@
 # Changelog
 
 ## [0.15.0](https://github.com/brave/browser-laptop/releases/v0.15.0dev)
- - Update muon to 2.58.6. ([#8161](https://github.com/brave/browser-laptop/issues/8161))
+ - Added Widevine support for Amazon Prime Video. ([#5233](https://github.com/brave/browser-laptop/issues/5233))
+ - Added tear off tabs - Ability to drag tab into new window. ([#4402](https://github.com/brave/browser-laptop/issues/4402))
+ - Added user pinning in Brave Payments. ([#7347](https://github.com/brave/browser-laptop/issues/7347))
+ - Added support for Back/forward gesture with three finger swipe. ([#7905](https://github.com/brave/browser-laptop/issues/7905))
+ - Added support for torrent link files. ([#6671](https://github.com/brave/browser-laptop/issues/6671))
+ - Added noscript into the URL bar. ([#5792](https://github.com/brave/browser-laptop/issues/5792))
+ - Added "Extensions..." to window menu. ([#8203](https://github.com/brave/browser-laptop/issues/8203))
+ - Added stop button for torrent downloads. ([#6768](https://github.com/brave/browser-laptop/issues/6768))
+ - Added extensions panel to about:preferences. ([#6530](https://github.com/brave/browser-laptop/issues/6530))
+ - Added a link to release notes in about:brave. ([#6130](https://github.com/brave/browser-laptop/issues/6130))
+ - Added context-menu option to close tabs in tab page. ([#5489](https://github.com/brave/browser-laptop/issues/5489))
+ - Added extension badge colors. ([#5367](https://github.com/brave/browser-laptop/issues/5367))
+ - Added text badges for extensions. ([#5366](https://github.com/brave/browser-laptop/issues/5366))
+ - Added selection ability for torrent files. ([#8148](https://github.com/brave/browser-laptop/issues/8148))
+ - Added temporary notice to sync prefs page regarding mobile sync and Beta label. ([#8121](https://github.com/brave/browser-laptop/issues/8121))
+ - Added a new tab page for private tabs. ([#7934](https://github.com/brave/browser-laptop/issues/7934))
+ - Added "Command+Click" to home button (opens home in a new background tab). ([#7718](https://github.com/brave/browser-laptop/issues/7718))
+ - Added origin display for downloads. ([#7468](https://github.com/brave/browser-laptop/issues/7468))
```

Comment:
Could also be important to note `Added origin for downloads, including unlock icon if download was insecure (HTTP)`